sql/parser/builtins.go

```diff
@@ -549,7 +549,19 @@ var builtins = map[string][]builtin{
 		}),
 	},
 
-	// TODO(thschroeter): implement round
+	"round": {
+		builtin{
+			returnType: DummyFloat,
+			types:      typeList{floatType, intType},
+			fn: func(args DTuple) (Datum, error) {
+				n := args[1].(DInt)
+				// TODO(thschroeter): bounds check on n
+				x := args[0].(DFloat)
+				return DFloat(roundDigits(float64(x),
+					int64(n))), nil
```

Comment:
The newline in the `return` statement seems excessive. Long lines can present readability problems, but so can excessively short ones.